README.md

```diff
@@ -45,9 +45,13 @@ from the broker. This processor must implement
 use Swarrot\Processor\ProcessorInterface;
 use Swarrot\Broker\Message;
 
-class Processor implements ProcessorInterface {
-    public function process(Message $message, array $options) {
+class Processor implements ProcessorInterface
+{
+    public function process(Message $message, array $options)
+    {
         echo sprintf("Consume message #%d\n", $message->getId());
+
+        return true; // Continue processing other messages
```

Comment:
Right now, a lot of processors (both in swarrot and in proprietary software) don't return a boolean (a lot of them return null).
Even if I'm not sure it's a good idea to enforce boolean as type hint for the return value (at least in the near future) it's still a good idea to add this in the documentation. :)
Maybe it will ease the migration for some people when we'll decide to add this type hint.